apps/frontend/src/theme/config.tsx

```diff
@@ -0,0 +1,75 @@
+const antdConfigGen = (isDarkMode: boolean) => {
+    const theme = styledComponentTheme(isDarkMode);
+
+    return {
+        token: {},
+        components: {
+            Button: {
+                defaultBg: theme.color.button.primary
+            },
+        }
+    };
+};
+
+export type StyledComponentTheme = {
```

Comment:
Why do you export this type?

Reply:
Actually seems not necessary for me too, will fix immediately